server/models/procedures.sql

```diff
@@ -639,18 +639,23 @@ BEGIN
       /*
         A positive remainder means that the debtor overpaid slightly and we should debit
         the difference to the debtor and credit the difference as a gain to the gain_account
-
-        - The debtor entity an invoice reference are not included on the gain
-          account transaction. In this case the debtor covered MORE than the 
-          invoiced amount and so referencing them on the enterprise gain will 
-          actually debit them the additional amount. 
       */
       IF (remainder > 0) THEN
-  
-        -- The debtor is not debited in this transaction. They have already 
-        -- balanced the invoice and their debt according to the invoice (the 
-        -- exact amount). The additional payment can just be put in a gain account.
-        
+
+        -- debit the debtor
+        INSERT INTO posting_journal (
```

Comment:
@jniles @lomamech Why are the comments I made removed? It looks to me like this was a problem with merging or rebasing branches.

As the removed comment points out writing this line causes the debt to the debtor to not balance. This will cause `master` to create inbalanced transactions. 

Please review the latest `master` file [procedures.sql](https://github.com/IMA-WorldHealth/bhima-2.X/blob/master/server/models/procedures.sql) to see what should be kept and removed here.